### ISSUE TITLE
npm dependencies for eslint-config-airbnb & eslint-plugin-jsx-a11y cannot be met

### ISSUE THREAD

**Issue by ankostis** [nteract/archived-desktop-app#583]
From yesterday that I'm trying to install `package.json` I'm getting this (note that I'm not experienced with npm):

```
+-- UNMET PEER DEPENDENCY eslint@3.2.2
+-- eslint-config-airbnb@9.0.1
| `-- UNMET PEER DEPENDENCY eslint@^2.9.0
`-- UNMET PEER DEPENDENCY eslint-plugin-jsx-a11y@2.0.1

npm ERR! peer dep missing: eslint@^2.9.0, required by eslint-config-airbnb@9.0.1
npm ERR! peer dep missing: eslint-plugin-jsx-a11y@^1.2.0, required by eslint-config-airbnb@9.0.1
npm ERR! peer dep missing: eslint@^2.9.0, required by eslint-config-airbnb-base@3.0.1
```

I tried to I upgrade `eslint-config-airbnb` from `9.0.1 --> 10.0.0` and `eslint-plugin-react: 5.2.2 --> 6.0.0`, the installation passes. 
Would that work?

**Comment by captainsafia** [comment]
Hello, @ankostis! Thanks for opening this issue. Let's see if we can figure out what's going on.

Let me try and reproduce the error on my machine and see if it occurs and if upgrading resolves the problem without causing any other issues.

To cover our bases, can you let me know what version of `npm` you are using and the commit hash of the version of master that you are on?

**Comment by captainsafia** [comment]
OK! I can reproduce the error but it's simply a warning on my end. Does this issue halt the `npm install` or are you able to complete it?

**Comment by ankostis** [comment]
No, it halts.
Note that I'm on Windows, and using `electron-prebuilt`, if that has any significance.

**Comment by ankostis** [comment]
I managed to built latest 0.0.5 under Windows; now the the outdated versions are just warnings.
I made a quick test build and I confirm that it was the merging of ~~5c8fdec~~ #582 that fixed it.

Still you may consider to upgrade the versions, not to receive the warning.
